requirements.txt

```diff
@@ -8,6 +8,9 @@ torch==1.8.1
 num2words==0.5.10
 word2number==1.1
 
+# for SpellCheckerPerturbation
+pyenchant==3.2.0
```

Comment:
This can't be integrated since the library used is not MIT/Apache Licensed. I would suggest to look for a replacement.